### PR TITLE
add $mille

### PR DESCRIPTION
**Pull request by 0xtokendeployer**

### CODE REVIEW

File: juno/assetlist.json [hunk 1]
```diff
@@ -1241,6 +1241,28 @@
...
+          "exponent": 0
+        },
+        {
+          "denom": "mile",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:juno1llg7q2d5dqlrqzh5dxv8c7kzzjszld34s5vktqmlmaaxqjssz43sxyhq0d",
+      "name": "Mille",
+      "display": "mille",
```
Comment (JeremyParish69) [inline juno/assetlist.json:1261]: note sure if the exp 6 denom is meant to be mile or mille, but the 'display' should match the exp-6 denom

File: juno/assetlist.json [hunk 2]
```diff
@@ -1241,6 +1241,28 @@
...
+      "symbol": "MILE",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/mmille.png"
```
Comment (JeremyParish69) [inline juno/assetlist.json:1264]: I think you got a typo on the file name 'mmille'

File: juno/assetlist.json (hunk 1 above)
Comment (0xtokendeployer) [reply juno/assetlist.json:1261]: Meant to be mile. Fixed the typo

**Review by 0xtokendeployer — COMMENTED** [review]
fixed typo in image url  + token display and denom fixed to "mile"

**Pull request merged**